content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md

```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
```

Comment:
```suggestion
```
Those people probably don't speak Brazilian Portuguese.

Reply:
This section can be fully removed.